examples/vignette.rb

```diff
@@ -74,5 +74,5 @@
 
 checkerboard = Image.read('pattern:checkerboard') { self.size = "#{ballerina.columns}x#{ballerina.rows}" }
 vignette = checkerboard[0].composite(ballerina, CenterGravity, OverCompositeOp)
-vignette.write('vignette.png')
+vignette.write('vignette1.png')
```

Comment:
@Watson1978 why was this renamed?

Reply:
https://github.com/rmagick/rmagick/blob/b6a448e5deb557ce794f75b2e65b4f4b537f2d4e/examples/vignette.rb#L65

It override the image.
I wanted to check both.